chia/util/generator_tools.py

```diff
@@ -64,3 +64,13 @@ def tx_removals_and_additions(npc_list: List[NPC]) -> Tuple[List[bytes32], List[
     additions.extend(additions_for_npc(npc_list))
 
     return removals, additions
+
+
+def list_to_batches(paths: List[Any], batch_size: int) -> Iterator[Tuple[int, List[Any]]]:
```

Comment:
this function should be easy to test now. I think we should have a unit test

Reply:
See 218292ffda35c7687d6cd3cbf5f7415688825e43.